centos_ci.sh

```diff
@@ -118,6 +118,8 @@ function upload_logs() {
   # http://stackoverflow.com/a/22908437/1120530; Using --relative as --rsync-path not working
   mkdir -p pr/$ghprbPullId/
   cp -R test/integration/out/test-results/* pr/$ghprbPullId/
+  # Change the file permission to 0644 so after rsync it can be readable by other user.
+  chmod 0644 $HOME/.crc/crc.log
```

Comment:
Wonder how it was working before! :-/

Reply:
@anjannath previously our file permission was different and after adding gosec we made those modifications.

Reply:
Ahh.. Okay, that makes sense!